code/computational_geometry/src/cohen_sutherland_lineclip/cohenSutherlandAlgorithm.cpp

```diff
@@ -127,7 +127,7 @@ void CohenSutherLandAlgo::cohenSutherland()
                 y2_ = y;
                 code2 = generateCode(x2_, y2_);
             }
-        }
+        } 
```

Comment:
What is the purpose of this?

Reply:
No, I was asking why you added an extra space at the end of line. Trailing whitespace adds nothing.

Reply:
Ohh, sorry for wrong interpretation, I will remove it.

Reply:
It belongs to indentation...

Reply:
How does the space after the brace belong to indentation? The spaces before the brace definitely to, but the trailing whitespace serves no purpose.

```suggestion
        }
```

Reply:
Yeah it is true I had wrong interpretation here... I will remove it surely.

Reply:
I had made the changes as per your suggestion please review it,